source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -333,21 +334,30 @@ this aggregation:
 .. code-block:: java
    :copyable: true
 
-   public static MqlNumber classAvg(MqlArray<MqlDocument> u) {
-       var sum = u.sum(x -> x.getInteger("finalGrade"));
-       var avg = sum.divide(u.size());
+   public static MqlNumber gradeAverage(MqlArray<MqlDocument> students, String fieldName) {
+       var sum = students.sum(student -> student.getInteger(fieldName));
+       var avg = sum.divide(students.size());
        return avg;
    }
-   
-   public static MqlString evalAvg(MqlNumber v) {
-       MqlNumber[] b = {of(70), of(85)};
-       var msg = v.switchOn(on -> on
-               .lte(b[0], m -> of("Needs improvement"))
-               .lte(b[1], m -> of("Meets expectations"))
+
+   public static MqlString evaluate(MqlNumber avg, MqlNumber[] benchmarks) {
+       var msg = avg.switchOn(on -> on
+               .lte(benchmarks[0], m -> of("Needs improvement"))
+               .lte(benchmarks[1], m -> of("Meets expectations"))
```

Comment:
```suggestion
       var message = grade.switchOn(on -> on
               .lte(benchmarks[0], g -> of("Needs improvement"))
               .lte(benchmarks[1], g -> of("Meets expectations"))
```
The `m` suggests that it corresponds to msg, but actually it corresponds to the grade that is being "switched on".